decidim-type/app/decorators/decidim/organization_decorator.rb

```diff
@@ -2,4 +2,6 @@
 
 Decidim::Organization.class_eval do
   has_many :types, -> { order(name: :asc) }, foreign_key: "decidim_organization_id", class_name: "DecidimType", inverse_of: :organization
+  has_many :users_with_any_role, -> { where.not(roles: []).or(where('"decidim_users"."id" in (select "decidim_participatory_process_user_roles"."decidim_user_id" from "decidim_participatory_process_user_roles")' +
```

Comment:
This decorator does not belong to this module.
Please, create a new decorator in `app/decorators` overriding this relation. Please, explain with a comment that this is an override.